tutorials/remove all entity shadows

```diff
@@ -0,0 +1,327 @@
+**Add / Remove all shadows by code**
+
+by Outlandishly Crafted updated by Jeremy Benisek AKA CyberAxe
+06212020
+
+#How to remove all entity shadows:
+
+Change these lines in the shadow.material file under the Resource Pack / materials folder. 
+NOTE: This folder is NOT included in the vanilla RP Pack examples and must be exported from a APK files or added by hand.
+NOTE: This has not been tested for blocks and has only been verified for entities. If you find it works on blocks too please let us know so we can add that in.
+
+Working shadow code: Shadows for ALL entities:
+"shadow_overlay": {
```

Comment:
All json code-blocks should be contained within proper json formatting. You can see the other markdown documents as an example. 

\```json

YOUR CODE HERE

\```